src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Extentions/StringExtention.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Exadel.CrazyPrice.Common.Extentions
+{
+    /// <summary>
+    /// Represents correcting string.
+    /// </summary>
+    public static class StringExtention
```

Comment:
Rename to StringExtenstions - probably one more to appear

Reply:
[Fix it](https://github.com/yegorl/exadel_discounts_be/blob/fix/src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Extentions/StringExtentions.cs)